ut/client_ut.cpp

```diff
@@ -712,7 +712,6 @@ TEST_P(ClientCase, DateTime64) {
 
             ASSERT_EQ(1U, block.GetColumnCount());
             if (auto col = block[0]->As<ColumnDateTime64>()) {
-                ASSERT_EQ(data.size(), col->Size());
                 for (size_t i = 0; i < col->Size(); ++i) {
                     EXPECT_EQ(data[i], col->At(i)) << " at index: " << i;
```

Comment:
Are the indices absolute? Or they are offsets within the current block?
If the latter, then the code still relies on the assumption that the entire data will be received in the [first] block. Maybe, technically it is what will always happen here, but if the data is set to size of, e.g., `100 000 000`, will this code still be correct?

Reply:
Fixed

Reply:
I mean, it it correct to use `i` that iterates over `col` ans an index for `data`?